### PR TITLE
[IMP] base_import: clarify error message in xls import

### PR DESCRIPTION
**Pull request by mart-e**
When an import failed because of a specific cell, the error was
   Error cell found while reading XLS/XLSX file: #N/A

Add the line and column index in the error for better debugging
Translate error message

Cherry-pick of odoo/odoo#30729 in master

cc @Yenthe666 as modifying the translation content and is only an improvement (no fix as we keep the message value in the end)

### CODE REVIEW

File: addons/base_import/models/base_import.py [hunk 1]
```diff
@@ -286,9 +286,11 @@ def _read_xls_book(self, book):
                     values.append(u'True' if cell.value else u'False')
                 elif cell.ctype is xlrd.XL_CELL_ERROR:
                     raise ValueError(
-                        _("Error cell found while reading XLS/XLSX file: %s") %
-                        xlrd.error_text_from_code.get(
-                            cell.value, "unknown error code %s" % cell.value)
+                        _("Invalid cell value at row %(row)s, column %(col)s: %(cell_value)s") % {
+                            'row': rowx + 1,
+                            'col': colx + 1,
```
Comment (Elkasitu) [inline addons/base_import/models/base_import.py]: `enumerate` can take a starting value other than `0` as a second parameter, e.g. `enumerate(row, 1)`

**Comment by Yenthe666** [comment]
Okay, I can live with that. :) Thanks for the retarget

**Comment by mart-e** [comment]
robodoo r+

**Comment by Yenthe666** [comment]
Ahh yes, this will be so helpful. Thanks! 🎉

**Pull request closed without merging**